src/providers/BasicPoolsProvider.tsx

```diff
@@ -44,20 +57,23 @@ type SharedSwapData = {
   miniChefRewardsPid: number | null
   isSynthetic: boolean
 }
+
 type MetaSwapInfo = SharedSwapData & {
   underlyingTokens: string[]
   basePoolAddress: string
   metaSwapDepositAddress: string
   underlyingTokenBalances: BigNumber[]
-  isMetaSwap: true
+  isMetaSwap: boolean
```

Comment:
This may mess things up downstream. Having this value be true for one type and false for another lets us do handy things like 
```
if (pool.isMetaSwap) {
  foo = pool.underlyingTokens.map(...)
}
```
without having to do `p?.underlyingTokens?.map() || []`

Reply:
I'll look into being able to add this back. It was causing me a ton of typing issues.